src/cli.ts

```diff
@@ -173,77 +173,74 @@ async function main() {
     result.links = filteredResults;
     console.log(JSON.stringify(result, null, 2));
     return;
-  } else if (format === Format.CSV) {
+  }
+  if (format === Format.CSV) {
     result.links = filteredResults;
     const csv = await toCSV(result.links);
     console.log(csv);
     return;
-  } else {
-    // Build a collection scanned links, collated by the parent link used in
-    // the scan.  For example:
-    // {
-    //   "./README.md": [
-    //     {
-    //       url: "https://img.shields.io/npm/v/linkinator.svg",
-    //       status: 200
-    //       ....
-    //     }
-    //   ],
-    // }
-    const parents = result.links.reduce((acc, curr) => {
-      const parent = curr.parent || '';
```

Comment:
While benign, I don't think this change makes the code any different or better.  Both `||` or `??` would be valid here, but there are reasons to use one over the other.  I may have a problem with this specific rule :)